playground/optimize-deps/dep-cjs-browser-field-bare/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "dep-cjs-browser-field-bare",
+  "private": true,
+  "version": "0.0.0",
+  "main": "index.js",
+  "browser": {
+    "events": "./events-shim.js"
```

Comment:
As a note to anyone else who's equally as curious in the future as me, *this* is the "bare id mapping" in the `browser` field.